package/lib/extractPreviewRefRepositoryName.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Accepts the host string and returns the host name.
+ *
+ * @param host - Host string
+ */
+const extractFirstSubdomain = (host: string): string => host.split(".")[0];
+
+/**
+ * Parses ref object from cookie and returns proper preview link
```

Comment:
This description isn't completely accurate; it could be something like: "Returns the repository name from an object-style Prismic ref."